### PR TITLE
[test-only] changing status code of the api tests

### PR DESCRIPTION
**Pull request by ScharfViktor**
https://github.com/owncloud/ocis/issues/3516 fixed but tests were failed because expected code was `400` instead of `409`

I changed code and deleted tests from expected failures list

### CODE REVIEW

File: tests/acceptance/features/apiGraph/createGroup.feature [hunk 1]
```diff
@@ -24,11 +24,11 @@ Feature: create group
     | Finance (NP)    |
     | slash\Middle    |
 
-  @issue-3516
+
```
Comment (saw-jan) [inline tests/acceptance/features/apiGraph/createGroup.feature:27]:
```suggestion
  @issue-3516
```

File: tests/acceptance/features/apiGraph/createGroupCaseSensitive.feature [hunk 2]
```diff
@@ -4,13 +4,13 @@ Feature: create groups, group names are case insensitive
   I want to create groups with similar cases
   So that I can check if the group names are case sensitive
 
-  @issue-3516
+
```
Comment (saw-jan) [inline tests/acceptance/features/apiGraph/createGroupCaseSensitive.feature:7]:
```suggestion
  @issue-3516
```

**Review by saw-jan — APPROVED** [review]
We can keep the issue tags for future reference (can be helpful if the scenario fails again)

LGTM :+1:

**Comment by ownclouders** [comment]
:boom: Acceptance test [<strong>localApiTests-apiGraph-ocis</strong>](https://drone.owncloud.com/owncloud/ocis/23332/51/1) failed. Further test are cancelled...

**Review by phil-davis — APPROVED** [review]

**Pull request merged**